<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScanDialog</class>
 <widget class="QDialog" name="ScanDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>392</width>
    <height>419</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scan Book</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="topSearchBarLayout">
       <item>
        <widget class="QLineEdit" name="searchTextEdit"/>
       </item>
       <item>
        <widget class="QPushButton" name="searchButton">
         <property name="text">
          <string>Search Online</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="clearButton">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QTableView" name="searchTable"/>
       </item>
       <item>
        <layout class="QFormLayout" name="informationLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="isbnLabel">
           <property name="text">
            <string>ISBN</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="isbnTextEdit"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="titleLabel">
           <property name="text">
            <string>Title</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="titleTextEdit"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="subtitleLabel">
           <property name="text">
            <string>Subtitle</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLineEdit" name="subtitleTextEdit"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="authorLabel">
           <property name="text">
            <string>Author</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLineEdit" name="authorTextEdit"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="publisherLabel">
           <property name="text">
            <string>Publisher</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLineEdit" name="publisherTextEdit"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="publishDateLabel">
           <property name="text">
            <string>Publish Date</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="publishDateTextEdit"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="pageCountLabel">
           <property name="text">
            <string>Page Count</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="pageCountTextEdit"/>
         </item>
         <item row="7" column="0">
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item row="8" column="0">
          <widget class="QLabel" name="languageLabel">
           <property name="text">
            <string>Language</string>
           </property>
          </widget>
         </item>
         <item row="8" column="1">
          <widget class="QLineEdit" name="languageTextEdit"/>
         </item>
         <item row="7" column="1">
          <widget class="QTextEdit" name="descriptionTextEdit"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="bottomButtonLayout">
       <item>
        <widget class="QProgressBar" name="progressBar">
         <property name="value">
          <number>24</number>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ScanDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>383</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ScanDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>386</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchButton</sender>
   <signal>clicked()</signal>
   <receiver>ScanDialog</receiver>
   <slot>requestBook()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>207</x>
     <y>197</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchTable</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>ScanDialog</receiver>
   <slot>setResults()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>135</x>
     <y>210</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>clearButton</sender>
   <signal>clicked()</signal>
   <receiver>ScanDialog</receiver>
   <slot>clearSearchField()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>343</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>searchTextEdit</sender>
   <signal>textChanged(QString)</signal>
   <receiver>ScanDialog</receiver>
   <slot>searchRequestAuto(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>111</x>
     <y>18</y>
    </hint>
    <hint type="destinationlabel">
     <x>195</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>requestBook()</slot>
  <slot>setResults()</slot>
  <slot>clearSearchField()</slot>
  <slot>searchRequestAuto(QString)</slot>
 </slots>
</ui>
